src/Map/SpatialId/api.ts

```diff
@@ -0,0 +1,24 @@
+// Ref: https://api-vt.geolonia.com/altitude.html
+const GEOID_HEIGHT_API = "https://api-vt.geolonia.com/api/altitude";
+
+type Result = {
+  lat: number;
+  lng: number;
+  geoid: number | "NaN";
+  altitude: number | "NaN";
+};
+
+export function getGeoidHeight(lng: number, lat: number): Promise<number | undefined> {
+  if (lat === undefined || lng === undefined) return Promise.resolve(undefined);
+  return fetch(`${GEOID_HEIGHT_API}?lat=${lat}&lng=${lng}`)
```

Comment:
I thought we should get it from user side, because I think there are some APIs to get the geoid height.
How do you think?

Reply:
Agree. Refactored on engine API.

Reply:
Update: now it's on geoid API.